frontend/packages/console-shared/src/components/formik-fields/item-selector-field/ItemSelectorField.tsx

```diff
@@ -68,40 +61,12 @@ const ItemSelectorField: React.FC<ItemSelectorFieldProps> = ({
     return null;
   }
 
-  const fieldId = getFieldId(name, 'selector');
+  const fieldId = getFieldId(name, 'itemselector');
   const isValid = !(selectedTouched && selectedError);
   const errorMessage = !isValid ? selectedError : '';
 
   return (
-    <FormGroup
-      fieldId={fieldId}
-      label={label}
-      helperTextInvalid={errorMessage}
-      isValid={isValid}
-      isRequired
-    >
-      {isRecommending && (
-        <>
-          <LoadingInline /> Detecting recommended {label}...
-        </>
-      )}
-      {recommended && (
-        <>
-          <Alert variant="success" title="Builder image(s) detected." isInline>
-            Recommended {label} are represented by{' '}
-            <StarIcon style={{ color: 'var(--pf-global--primary-color--100)' }} /> icon.
-          </Alert>
-          <br />
-        </>
-      )}
-      {couldNotRecommend && (
-        <>
-          <Alert variant="warning" title="Unable to detect the builder image." isInline>
-            Select the most appropriate one from the list to continue.
-          </Alert>
-          <br />
-        </>
-      )}
+    <FormGroup fieldId={fieldId} helperTextInvalid={errorMessage} isValid={isValid} isRequired>
```

Comment:
Missing  optional `label` prop.
You may not use it now but as a re-usable field, it should be present.